tests/misc.cpp

```diff
@@ -1220,6 +1220,10 @@ get_solver_type(char* s)
     ret = QUDA_CG3NR_INVERTER;
   } else if (strcmp(s, "ca-cg") == 0){
     ret = QUDA_CA_CG_INVERTER;
+  } else if (strcmp(s, "ca-cgne") == 0){
+    ret = QUDA_CA_CGNE_INVERTER;
+  } else if (strcmp(s, "ca-cgnr") == 0){
+    ret = QUDA_CA_CGNR_INVERTER;
   } else if (strcmp(s, "ca-gcr") == 0){
```

Comment:
clang-format suggested the following change:


```
-  } else if (strcmp(s, "ca-gcr") == 0){
+  } else if (strcmp(s, "ca-gcr") == 0) {
```

<!-- identifier: clang-format -->
<!-- File: tests/misc.cpp -->